### PR TITLE
Add .editorconfig for issue #68

### PR DESCRIPTION
**Pull request by toptalo**
* add .editorconfig
* reformat code according .editorconfig
* add IntelliJ IDEA section to .gitignore
* fix Typos:
  - `Sans-Serif` to lowercase
  - `tranparent` to `transparent`

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,22 @@
+; This file is for unifying the coding style for different editors and IDEs
+; See editorconfig.org
+
+; top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
```
Comment (ummahusla) [inline .editorconfig]: Can we switch this to use `tabs`

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+indent_size = 4
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{css,html,js,json}]
+indent_size = 2
+
+[*.{scss,less}]
```
Comment (ummahusla) [inline .editorconfig]: This  can be `2` instead of `4`

**Review by ummahusla — CHANGES_REQUESTED** [review]

**Comment by ummahusla** [comment]
@toptalo Thanks for your PR, can you please update the requirements

File: .editorconfig (hunk 1 above)
Comment (toptalo) [reply .editorconfig]: Of course, I did as it was now

File: .editorconfig (hunk 2 above)
Comment (toptalo) [reply .editorconfig]: Is it correct: all files must have tabs as indent. Indent size = 2?

Comment (ummahusla) [reply .editorconfig]: Yeah, I guess you make the global size equal to `2`. I'm okay with that.

**Comment by toptalo** [comment]
I updated requirements

**Comment by ummahusla** [comment]
@toptalo Thanks! 👍

**Pull request merged**